### PR TITLE
alex's_solution

### PR DESCRIPTION
**Pull request by ascreven**
comfort: 4
completeness: 5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,9 +39,9 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```Scope is where a variable can be called. The scope of an element can either be global or local. Global variables are defined outside a function (or inside a function without a var before their name) can be accessed anywhere in the script. Local variables are defined inside functions and only have their set value in the function in which they are defined.
```
Comment (mattscilipoti) [inline README.md:42]: :+1: 
Locally scoped variables are also available within contained functions.  This is sometimes referred to as "chaining of scopes": Any function defined within another function has a local scope which is linked to the outer function.

More info: http://www.coolcoder.in/2014/03/everything-you-need-to-know-about.html

File: README.md [hunk 2]
```diff
@@ -51,8 +52,16 @@ the variable pizza inside your method.
...
+
 // write code here
+var pizza = {
+  temperature: 70,
+  bake: function (){
+    this.temperature = 300
+  }
+}
 ```
+-->
```
Comment (mattscilipoti) [inline README.md:64]: I'm curious.  What are these arrows for?

File: README.md [hunk 3]
```diff
@@ -101,23 +120,30 @@ Which of the following statements will work, assuming jQuery is loaded?
...
 ```
-[] `$(".post").css("background", "peachpuff")`
+[x] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.getElementsByClassName("post")[0].innerHTML`
+[x] `$(".post").html()`
+[x] `document.getElementsByClassName("post")[0].innerHTML`
 [] `document.getElementsByClassName("post").innerHTML`
 ```
```
Comment (mattscilipoti) [inline README.md:128]: :tada:

File: README.md [hunk 4]
```diff
@@ -16,8 +17,8 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
-```
+```clearfix is used to style information in boxes by giving the box an actual height so the information fits in it properly. One way you can do this is but using the psuedo-element :after. This creates a blank element that has height that the inner content can float in.
+``` -->
```
Comment (mattscilipoti) [inline README.md:21]: :+1:

File: README.md [hunk 5]
```diff
@@ -51,8 +52,16 @@ the variable pizza inside your method.
...
 Your Answer:
 ```js
+
 // write code here
+var pizza = {
+  temperature: 70,
+  bake: function (){
+    this.temperature = 300
+  }
+}
```
Comment (mattscilipoti) [inline README.md:62]: :+1:

File: README.md [hunk 6]
```diff
@@ -65,7 +74,17 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
...
+
+function createAlert(arg) {
+  alert(arg)
+}
+function doSomthing(thingToDo) {
+  thingToDo('it works')
+}
+doSomthing(createAlert)
+
 ```
```
Comment (mattscilipoti) [inline README.md:86]: :+1:

File: README.md [hunk 7]
```diff
@@ -87,7 +106,7 @@ Which of the following represents a correct workflow for submitting a PR on a no
 
 Select 1:
 ```
-[] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
+[x] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
```
Comment (mattscilipoti) [inline README.md:109]: :+1:

File: README.md [hunk 8]
```diff
@@ -101,23 +120,30 @@ Which of the following statements will work, assuming jQuery is loaded?
...
 
 Your Answer:
 ```js
-// your code here
+write code here
+
+<button id=greeting>push me</button>
+
+$('#greeting').on('click', function(){
+  $('<p>hello</p>').appendTo($('body'))
```
Comment (mattscilipoti) [inline README.md:143]: :+1:  
I had to lookup `appendTo`.  I usually use `$('body').append($('<p>hello</p>');`.  Either works.

File: README.md [hunk 9]
```diff
@@ -131,5 +157,5 @@ role, goal, and reason for each.
 Finally, link to your repo on github in the space below.
 
 Your Answer:
-```text
+https://github.com/ascreven/Guess-the-Location
```
Comment (mattscilipoti) [inline README.md:160]: :+1:

File: index.html [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+</body>
+
+</html>
+
+<script src='https://code.jquery.com/jquery-2.1.4.js'></script>
+<script>
+$('#greeting').on('click', function(){
+  $('<p>hello</p>').appendTo($('body'))
+})
+</script>
```
Comment (mattscilipoti) [inline index.html:18]: :tada: above and beyond, but good to see.

**Comment by mattscilipoti** [comment]
Looks great.  I had a couple of clarifications or additions.
Please review the answer key, in the [solution branch](https://github.com/ga-dc/wdi7-quiz2/tree/solution)

File: README.md (hunk 2 above)
Comment (ascreven) [inline README.md:64]: I accidentally messed up the formatting of the document. They don't mean anything though

Comment (mattscilipoti) [inline README.md:64]: thx

**Pull request closed without merging**